tools/python/transit/gtfs/download_gtfs.py

```diff
@@ -9,6 +9,7 @@
 import json
 import logging
 import os
+import pandas
```

Comment:
You use Pandas lib to parse CSV file. It adds extra dependency. While there is default Python 3 library [csv](https://docs.python.org/3.8/library/csv.html).

Reply:
Pandas is currently being used by another file in the project (/search/search_quality/scoring_model.py), so I was of the opinion that the module will already be installed to run the project, i.e. not an extra dependency.
I will look into a csv module alternative.

Reply:
Our general rule: fewer dependencies are better than more. We introduce new 3party dependencies ONLY if there is no other comparable way to solve the problem.

Reply:
I have changed the code to work with the default csv module :)